### DISCUSSION TITLE
I would run my Adonisjs App on local machine

### DISCUSSION THREAD

**Discussion by Daghsny** [adonisjs/core#4500] [Help]
After I finished my adonisJs app V5 with sql database, everything worked fine with vs code in linux. Now i would like to use my app on a local machine in windows.
My question is after making a build how to run this app and how to connect it with the sql.
Please help me

**Comment by GabrielNBDS** [comment]
Copy the .env file from your project to ./build, then change the database vars in there to the credentials of your desired database. To run it simply run `node ./build/server.js`
